packages/styles/scss/components/button/_button.scss

```diff
@@ -21,6 +22,98 @@
 @use '../../utilities/visually-hidden' as *;
 @use '../../utilities/layout';
 
+@mixin button-set-fluid-column() {
+  --flex-direction: column;
+
+  .#{$prefix}--btn {
+    flex: initial;
+    inline-size: 100%;
+    max-inline-size: none;
+  }
+
+  .#{$prefix}--btn--ghost {
+    padding-inline-start: $spacing-05;
+  }
+}
+
+@mixin button-set-fluid-layout() {
+  container-name: button-set;
+  container-type: inline-size;
+
+  .#{$prefix}--btn-set__fluid-inner {
+    --flex-direction: row;
+
+    display: flex;
+    flex-direction: var(--flex-direction);
+    align-items: stretch;
+    justify-content: flex-end;
+    inline-size: 100%;
+  }
+
+  /* NOTE: Container queries are based on this value */
+  $min-inline-button-size: convert.to-rem(176px);
```

Comment:
Is this defined in the spec, or is it derived from just the example buttons? Does it need to be configurable? Is there a non-magic number we could use instead?